src/pal/src/arch/i386/asmconstants.h

```diff
@@ -63,14 +65,13 @@
 #define CONTEXT_Xmm13 CONTEXT_Xmm12+16
 #define CONTEXT_Xmm14 CONTEXT_Xmm13+16
 #define CONTEXT_Xmm15 CONTEXT_Xmm14+16
-#define CONTEXT_VectorRegister CONTEXT_Xmm15+16
-#define CONTEXT_VectorControl CONTEXT_VectorRegister+16*26
-#define CONTEXT_DebugControl CONTEXT_VectorControl+8
+#define CONTEXT_Ymmh CONTEXT_FltSave+FLOATING_SAVE_AREA_SIZE
+#define CONTEXT_DebugControl CONTEXT_Ymmh+16*16
 #define CONTEXT_LastBranchToRip CONTEXT_DebugControl+8
 #define CONTEXT_LastBranchFromRip CONTEXT_LastBranchToRip+8
 #define CONTEXT_LastExceptionToRip CONTEXT_LastBranchFromRip+8
 #define CONTEXT_LastExceptionFromRip CONTEXT_LastExceptionToRip+8
-#define CONTEXT_Size CONTEXT_LastExceptionFromRip+8
+#define CONTEXT_Size CONTEXT_LastExceptionFromRip+8+8 // another +8 for 16-byte alignment
```

Comment:
It seems that if you fix the CONTEXT_DebugControl offset, this will not be necessary.